src/component/checkboxes.html

```diff
@@ -0,0 +1,16 @@
+<template>
+
+  <template containerless repeat.for="option of options">
+    <div class="checkbox">
```

Comment:
This className is pretty generic. It might cause CSS issues in real-world applications.